daphne/src/lib.rs

```diff
@@ -304,6 +305,23 @@ pub struct DapGlobalConfig {
     pub supported_hpke_kems: Vec<HpkeKemId>,
 }
 
+impl DapGlobalConfig {
+    /// Generate a list of HPKE receiver configurations, one for each element of supported KEM
+    /// algorithm. `first_config_id` is used as the first config ID; subsequent IDs are chosen by
+    /// incrementing `first_config_id`.
```

Comment:
Reviewer note: The reason to choose the initial id at random is to prevent peers from ossifying on fixed ids. We could consider choosing random, distinct IDs, but I don't think this is necessary.